tests/perfs/go.mod

```diff
@@ -0,0 +1,184 @@
+module github.com/mezo-org/mezod/tests/perfs
```

Comment:
```suggestion
module github.com/mezo-org/mezod/tests/performance
```

Reply:
addressed in dce6efb18c0e670c24f97a549e268d689437ef02